### PR TITLE
Remove obsolete japicmp exclude, use latest patch version.

### PR DESCRIPTION
**Pull request by tweise**
@vrozov please review

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -367,7 +367,7 @@
         <plugin>
           <groupId>com.github.siom79.japicmp</groupId>
           <artifactId>japicmp-maven-plugin</artifactId>
-          <version>0.7.0</version>
+          <version>0.7.2</version>
```
Comment (vrozov) [inline pom.xml]: Upgrade to 0.9.0? There are few bug fixes in 0.8.x and 0.9.0 that I needed in netlet.

Comment (tweise) [reply pom.xml]: Is it backward compatible?

Comment (vrozov) [reply pom.xml]: I guess yes based on minor version change, but why does it matter?

Comment (tweise) [reply pom.xml]: Anything < 1.0.0 does not come with such guarantee.

**Comment by tweise** [comment]
updated to use 0.9.0

**Pull request merged**